neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -107,8 +107,32 @@ public void CalculateFees()
             }
             using (Snapshot snapshot = Blockchain.Singleton.GetSnapshot())
             {
+                // Compute verification gas
```

Comment:
Should compute for all cosigners.

Reply:
Where is the cosigner's? i thought that we remove this in the past 🤔

Reply:
We "removed" it in #784. But it has not been accepted yet.

Reply:
Sorry, but i can find only one signature, the `Witness` , and inside the `TransactionAttributes` there are only `Url` ...

Reply:
Sorry, it was removed, my mind was stuck. . .